editor/editor_audio_buses.cpp

```diff
@@ -500,6 +500,10 @@ void EditorAudioBus::_effect_selected() {
 	updating_bus = false;
 }
 
+void EditorAudioBus::_effects_deselected() {
+	effects->deselect_all();
```

Comment:
This works for some cases when focus is lost, but it's kind of annoying when your mouse is just hovering over another node like the scene editor. Is there a way to check "is left mouse button down" at this part?

Reply:
No. Other docks don't deselect on focus lost (e.g. file dock), rather they do it when you click away. Though then there is inspector, which doesn't allow deselecting properties 🤷‍♂️

You could use "empty_clicked" signal for deselecting. This means you'd need to click the empty space under effects, but they'd be deselectable in a predictable way at least. The empty space in bus layout (between buses) could be used too.

Reply:
Oh okay, the empty click event makes more sense. I'll try that, thanks! I couldn't find a generic one for "clicked somewhere else /outside/ of the current panel," so I guess that doesn't exist?

Reply:
Yep, clicking another control at most can steal focus.